main.py

```diff
@@ -0,0 +1,87 @@
+import operator
+from typing import Callable
+Self = 'Matrix'
+
+
+# Purpose: Create a class to represent a two-dimensional array
+class Matrix:
+    """A class to represent a two-dimensional array"""
+    def __init__(self, matrix: tuple[tuple[float, ...], ...]):
+        self._matrix = matrix
+
+    def __str__(self):
+        """Return a string representation: ((1, 2), (3, 4))"""
+        return str(self._matrix)
+
+    def __repr__(self):
+        """ return: a string representation: Matrix(((1, 2), (3, 4)))"""

```

Comment:
the convention is 'Return a string representation'
I see that you want to show an example, there is a way of writing testable comments that represent the function's action, please use that (here and in all other relevant function in the class)